packages/terra-hyperlink/tests/jest/Hyperlink.test.jsx

```diff
@@ -6,59 +6,59 @@ import Hyperlink from '../../src/Hyperlink';
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render a document hyperlink', () => {
-    const wrapper = render(<Hyperlink href="https://www.cerner.com" variant="document">Document hyperlink</Hyperlink>);
+    const wrapper = render(<Hyperlink text="Document hyperlink" href="https://www.cerner.com" variant="document" />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render a external hyperlink', () => {
-    const wrapper = render(<Hyperlink href="https://www.cerner.com" variant="external">External hyperlink</Hyperlink>);
+    const wrapper = render(<Hyperlink text="External hyperlink" href="https://www.cerner.com" variant="external" />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render a image hyperlink', () => {
-    const wrapper = render(<Hyperlink href="https://www.cerner.com" variant="image">Image hyperlink</Hyperlink>);
+    const wrapper = render(<Hyperlink text="Image hyperlink" href="https://www.cerner.com" variant="image" />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render a underline hidden hyperlink', () => {
-    const wrapper = render(<Hyperlink href="https://www.cerner.com" isUnderlineHidden>Underline hidden hyperlink</Hyperlink>);
+    const wrapper = render(<Hyperlink text="Underline hidden hyperlink" href="https://www.cerner.com" isUnderlineHidden />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render a video hyperlink', () => {
-    const wrapper = render(<Hyperlink href="https://www.cerner.com" variant="video">Video hyperlink</Hyperlink>);
+    const wrapper = render(<Hyperlink text="Video hyperlink" href="https://www.cerner.com" variant="video" />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render with custom props', () => {
-    const wrapper = render(<Hyperlink href="https://www.cerner.com" target="_self" rel="nofollow">Custom props hyperlink</Hyperlink>);
+    const wrapper = render(<Hyperlink text="Custom props hyperlink" href="https://www.cerner.com" target="_self" rel="nofollow" />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render with custom props with external variant', () => {
-    const wrapper = render(<Hyperlink href="https://www.cerner.com" target="_parent" rel="external" variant="external">Custom target and rel props external hyperlink</Hyperlink>);
+    const wrapper = render(<Hyperlink text="Custom target and rel props external hyperlink" href="https://www.cerner.com" target="_parent" rel="external" variant="external" />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it('correctly applies the theme context className', () => {
     const wrapper = mount(
       <ThemeContextProvider theme={{ className: 'orion-fusion-theme' }}>
-        <Hyperlink href="https://www.cerner.com">Default hyperlink</Hyperlink>
+        <Hyperlink text="Default hyperlink" href="https://www.cerner.com" />
```

Comment:
What is the thinking behind changing from using children to using the text prop? Is there some benefit we get in doing so?

Reply:
Actual requirement was to have the prop as a string type and required  and then maybe the text sounded better than children.